README.rst

```diff
@@ -66,3 +66,4 @@ Reporting Issues
 To report an issue with Salt, please follow the guidelines for filing bug reports:
 
 * http://docs.saltstack.com/en/latest/topics/development/reporting_bugs.html
+ping
```

Comment:
Actually only salt admins can retrigger tests using this method, but contributors can retrigger tests by closing and repoening them.